common/static/common/templates/discussion/new-post-alert.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="alert alert-error pattern-library-shim" role="alert" aria-labelledby="alert-title-error" tabindex="-1">
+    <span class="icon alert-icon fa fa-warning pattern-library-shim" aria-hidden="true"></span>
```

Comment:
Do you need the pattern-library-shim class at every level? I thought it could just be included on the outermost div?